69.md

```diff
@@ -0,0 +1,205 @@
+New NIP - Defined vocabulary for NIP-36 and NIP-56
+
+(NIP-69?)
+
+Context:
+
+To be used effectively all forms of content reporting - whether self-reporting (via NIP-36) or 3rd party reporting (via NIP-56) need to be aligned in how they operate. A common, translatable "vocabulary" to use in those reports is the first step in aligning their use.
+
+These categories and contexts represent the types of content that relay owners and sites hosting client apps may need to filter to be compliant with local laws or to tailor the content on their site or relay to the standards of the community they support.
+
+
+
+Disclaimer:
+
+The fact that something is on this list says nothing about the ethical or moral quality of the content.
+
+A NIP-36 or NIP-56 report is not a suggestion to censor content, nor are they endorsements of censorship. Nostr is specifically designed to prevent any mandate to use NIP-36 or NIP-56 data. The data exists simply for contexts where it is needed or desired. Apps that support NIP-36 and NIP-56 reporting do not need to support all of the types and contexts listed below or present them to users. Likewise it's likely that many relays will instruct their moderators to ignore all the report types except those which pose serious legal issues for their organization.
+
+Some of the content types involve "in the eye of the beholder" judgement calls. Relay owners and Nostr-based communities will use and/or ignore reports only as the reports meet their particular standards. There won't be a single standard on any of these items, but rather thousands of standards.
+
+To reiterate - there is no way to censor Nostr. This isn't about censorship. It's about facilitating the content moderation necessary to be compliant with local laws and community standards that define the community of users for a particular relay or client app.
+
+
+
+Reportable Types:
+
+These types expand and standardize the "report types" defined in NIP-56. Existing NIP-56 report types (shown below as "sometype | ...") should be supported by clients for backwards compatibility, but should not otherwise be used in future development.
```

Comment:
### Reportable Types

These types expand and standardize the `report type` defined in NIP-56. Existing NIP-56 report types (`Old Code` in table below) should be supported by clients for backwards compatibility, but should not otherwise be used in future development.